### PR TITLE
Issue/182 normalised gsp data

### PR DESCRIPTION
**Pull request by peterdudfield**
# Pull Request

## Description

- get installed capacity from pv live for every gsp at every datetime
- use this data to normalise
- update script to get raw data
- update test data

Fixes issue #182 

## How Has This Been Tested?

- add unittests
- checked raw data  has maximum <=1 - TODO on full set

- [ ] No
- [x] Yes

## Checklist:

- [x] My code follows [OCF's coding style guidelines](https://github.com/openclimatefix/nowcasting/blob/main/coding_style.md)
- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

File: tests/data_sources/gsp/test_gsp_metadata.py [hunk 1]
```diff
@@ -73,52 +69,3 @@ def test_get_pv_gsp_shape_from_eso():
...
-
-    start = datetime(2019, 1, 1, tzinfo=pytz.utc)
-    end = datetime(2019, 1, 2, tzinfo=pytz.utc)
-
-    gsp_pv_df = load_pv_gsp_raw_data_from_pvlive(start=start, end=end, number_of_gsp=10)
-
-    assert isinstance(gsp_pv_df, pd.DataFrame)
-    assert len(gsp_pv_df) == (48 + 1) * 10
-    assert "datetime_gmt" in gsp_pv_df.columns
-    assert "generation_mw" in gsp_pv_df.columns
```
Comment (peterdudfield) [inline tests/data_sources/gsp/test_gsp_metadata.py:124]: this are just moved to the file below

**Review by JackKelly — APPROVED** [review]
LGTM!  Good work!

**Review by jacobbieker — APPROVED** [review]
Looks good!

**Comment by peterdudfield** [comment]
I'm just running the script to make the raw data - I think i'll wait unil that is done, before merging. Just so I dont break things my merging

**Pull request merged**